templates/onboarding/index.html

```diff
@@ -13,17 +13,39 @@
     </a>
   </div>
 
-  <!-- Text container -->
-  <div class="mt-8 mb-4 text-center">
-    <h2 class="text-dark" style="font-size:2rem">Let's take back control of your inbox!</h2>
+
+  <div id="loading-section" class="mt-8 mb-4 text-center">
+    <h3>Loading...</h3>
   </div>
 
-  <!-- Button container -->
-  <div class="mt-8 text-center">
-    <a class="mx-6 p-4 text-decoration-none" style="background:black;color:white;" href="{{ url_for('auth.register', next=url_for('onboarding.final')) }}">Create a new account</a>
-    <a class="mx-6 p-4 text-decoration-none" style="background:white;color:black;border-radius: 2px;border:1px solid black;" href="{{ url_for('auth.login', next=url_for('onboarding.final')) }}">I already have an account</a>
+  <div id="content-section" style="display: none">
```

Comment:
I wonder if we can test whether the user is authenticated here (ie `current_user is not None`) and either display the login/register button or send the PERFORM_EXTENSION_SETUP message? This way we don't need to show/hide the login/register button in JS.

Reply:
if the user is authenticated and we send the message, what should be the contents of the page? It could be that a user ends up here without having the extension installed. Do we want to wait for a couple secs and then redirect them to the dashboard?

Reply:
> if the user is authenticated and we send the message, what should be the contents of the page

I think we can show the message "Setting up SimpleLogin browser extension ..." in this case. 

> Do we want to wait for a couple secs and then redirect them to the dashboard?

No I think we should redirect them to the final onboarding page. This case happens for user who is already logged in in SL and now install the extension. We want them to start using the extension.

Reply:
Changed it.

* If the user is logged in, we show the same message as the `setup_done.html` template and send the event.
* If the user is not logged in, show the buttons.